pom.xml

```diff
@@ -105,6 +100,41 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Uncomment mongodb-v3-driver to test the v3 driver with migrations -->
+<!--        <dependency>-->
+<!--            <groupId>io.mongock</groupId>-->
+<!--            <artifactId>mongodb-v3-driver</artifactId>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+
+        <!-- Uncomment mongodb-sync-v4-driver to test the v4 driver with migrations -->
+<!--        <dependency>-->
+<!--            <groupId>io.mongock</groupId>-->
+<!--            <artifactId>mongodb-sync-v4-driver</artifactId>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+
+        <!-- Uncomment mongodb-springdata-v2-driver and spring-data-mongodb to test the spring data migrations -->
```

Comment:
Should this one actually say to "comment" it as opposed to "uncomment"?

Reply:
But, the mongodb-springdata-v2-driver is currently _not_ commented...?

Reply:
That is correct, maybe I just add a comment higher that says "Comment/Uncomment the below drivers to ensure you only have the one you want for testing purposes"

Reply:
ok